worlds/tunic/__init__.py

```diff
@@ -373,7 +385,8 @@ def fill_slot_data(self) -> Dict[str, Any]:
             "Hexagon Quest Holy Cross": self.ability_unlocks["Pages 42-43 (Holy Cross)"],
             "Hexagon Quest Icebolt": self.ability_unlocks["Pages 52-53 (Icebolt)"],
             "Hexagon Quest Goal": self.options.hexagon_goal.value,
-            "Entrance Rando": self.tunic_portal_pairs
+            "Entrance Rando": self.tunic_portal_pairs,
+            "disable_local_spoiler": max(int(tunic_settings["disable_local_spoiler"]), int(self.multiworld.is_race)),
```

Comment:
Just making sure - should this really be an int? (`max` also works for bools.)

Reply:
Actually the self.settings mentioned above should be used here like so: `self.settings.disable_local_spoiler`

Reply:
It's getting sent to C#, so for consistency we're converting them to ints before it gets over to the mod. I know we could just convert it on the mod-side, but it's arbitrary either way so might as well do it here for consistency.

Reply:
Sorry, was busy today but I'll fix the settings stuff in a little bit